backend/src/migrations/20240227113506_add_workouts_table.ts

```diff
@@ -0,0 +1,69 @@
+import { type Knex } from 'knex';
+
+const TABLE_NAME = 'workouts';
+const USERS_TABLE_NAME = 'users';
+
+const ColumnName = {
+    ID: 'id',
+    USER_ID: 'user_id',
+    ACTIVITY: 'activity',
+    STEPS: 'steps',
+    HEART_RATE: 'heart_rate',
+    WORKOUT_STARTED_AT: 'workout_started_at',
+    WORKOUT_ENDED_AT: 'workout_ended_at',
+    DISTANCE: 'distance',
+    SPEED: 'speed',
+    KILOCALORIES: 'kilocalories',
+    CREATED_AT: 'created_at',
+    UPDATED_AT: 'updated_at',
+};
+
+const ACTIVITY_ENUM = `${ColumnName.ACTIVITY}_enum`;
```

Comment:
Looks like we will have duplicated enum because in achievements migration we have activity_type_enum. I think you should rename **activity** to **activity_type** to match the name in achievements and goals table